### ISSUE TITLE
BaseRequestFilter GET filter

### ISSUE THREAD

**Issue by woot4moo** [acesinc/convergent-ui#9]
The GET request filtering is invalid for certain large GET requests that must be sent as a POST.

**Comment by andrewserff** [comment]
Are you asking for Convergent UI support of POST requests?

**Comment by woot4moo** [comment]
No.  I am suggesting that certain large GET requests would be sent as a
POST, this in turn could result in lost filtering.

On Dec 5, 2016 12:26, "Andrew Serff" <notifications@github.com> wrote:

Are you asking for Convergent UI support of POST requests?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/acesinc/convergent-ui/issues/9#issuecomment-264918007>,
or mute the thread
